pkg/conversion/mapi2capi/aws_test.go

```diff
@@ -207,10 +207,8 @@ var _ = Describe("mapi2capi AWS conversion", func() {
 					EBS: &mapiv1.EBSBlockDeviceSpec{},
 				}}),
 			),
-			infra: infra,
-			expectedErrors: []string{
-				"spec.providerSpec.value.blockDevices[0].ebs.volumeSize: Required value: volumeSize is required, but is missing",
-			},
+			infra:            infra,
+			expectedErrors:   []string{}, // No error is expected anymore, because volumeSize is now set to a default amount if nil.
```

Comment:
If I'm thinking about this the right way, it seems like this error shouldn't be expected anymore, because if the volumeSize is missing it should now be set (and this test is no longer spitting back this expected error). I don't really know if it's helpful to keep this test case here in this state? Is there a more useful way we could update this?